### PR TITLE
[Raid]Fix TBC Zul'aman hostage chest unlootble

### LINKED ISSUE #9155: TBC ZAM 10 people's instance can't get the box rewards
### Current Behaviour

TBC ZAM after kill the boss, the first boss's box locked can not open, the 2-4 boss's the box can not click.

### Expected Blizzlike Behaviour

The box should be able to open and pick up directly after killing the boss in the time limit and after talk with the npc.

### Source

_No response_

### Steps to reproduce the problem

1.kiss the first boss, and talk with the npc, the box apper but locked
2. kiss 2-4 boss, and talk with the npc, the box apper but can't click to.

### Extra Notes

_No response_

### AC rev. hash/commit

AzerothCore rev. 1cd8c43447c3 2021-11-10 10:18:40 +0100 (master branch) (Unix, RelWithDebInfo, Static)
Using SSL version: OpenSSL 1.1.1f  31 Mar 2020 (library: OpenSSL 1.1.1f  31 Mar 2020)
Using Boost version: 1.71.0
Using MySQL version: 80027
Using CMake version: 3.16.3
Compiled on: Linux 5.4.0-77-generic
Worldserver listening connections on port 54345
Realmlist (Realm Id: 1) configured in port 54345
VMAPs status: Enabled. LineOfSight: 1, getHeight: 1, indoorCheck: 1
MMAPs status: Enabled
maps directory located in /root/azeroth-server/bin/maps. Total size: 252191207 bytes
vmaps directory located in /root/azeroth-server/bin/vmaps. Total size: 658130721 bytes
mmaps directory located in /root/azeroth-server/bin/mmaps. Total size: 2196365032 bytes
Using enUS DBC Locale as default. All available DBC locales: enUS 
Using World DB: ACDB 335.5-dev
Using World DB Revision: 2021_11_10_02
Using Character DB Revision: 2021_10_14_00
Using Auth DB Revision: 2021_10_14_00
LoginDatabase queue size: 0
CharacterDatabase queue size: 0
WorldDatabase queue size: 0

### Operating system

ubuntu 20.4 x64

### Custom changes or Modules

mod-1v1-arena
mod-ah-bot
mod-autobalance
mod-auto-revive
mod-cfbg
mod-custom-login
mod-eluna-lua-engine
mod-learn-spells
mod-multi-client-check
mod-npc-all-mounts
mod-npc-beastmaster

### PR DESCRIPTION
**Pull request by betayi**
## Changes Proposed:
-  continue #9378 
-  

## Issues Addressed:

- Closes #9155 

## Tests Performed:

- 
- 

## How to Test the Changes:

1.enter zam talk door keepera open door and start hostage rescue
2.kill any 3 of boss  Akil'zon Jan'alai Nalorakk and Halazzi
3.the hostage summoned a chest
4.loot it

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by temperrr** [comment]
Hey!

Thanks for your PR! What is the status (WIP, Ready to be reviewed/tested)?

**Comment by betayi** [comment]
> Hey!
> 
> Thanks for your PR! What is the status (WIP, Ready to be reviewed/tested)?

I ‘m tested， but it seems  the status flag was not changed by me ...

File: data/sql/updates/pending_db_world/rev_1637978286218341866.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1637978286218341866');
+
+delete from gameobject_template_addon where entry in (186648,187021,186672,186667);
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1637978286218341866.sql:3]:
```suggestion
DELETE FROM `gameobject_template_addon` WHERE `entry` IN (186648,187021,186672,186667);
```

File: data/sql/updates/pending_db_world/rev_1637978286218341866.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+insert into gameobject_template_addon values
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1637978286218341866.sql:4]:
```suggestion
INSERT INTO `gameobject_template_addon` (`entry`, `faction`, `flags`, `mingold`, `maxgold`) VALUES
```

**Review by Nefertumm — CHANGES_REQUESTED** [review]
Please, address the changes I made

**Review by Nefertumm — APPROVED** [review]

**Comment by betayi** [comment]
same issue with #9155

**Comment by Voxstrasza** [comment]
I tested this.  3 chests from 3 bosses.  Video:

https://youtu.be/xT8PIotJkiY

**Pull request merged**